src/stream/src/executor/over_window/over_partition.rs

```diff
@@ -352,7 +371,7 @@ impl<'a, S: StateStore> OverPartition<'a, S> {
 
     /// Find all ranges in the partition that are affected by the given delta.
     /// The returned ranges are guaranteed to be sorted and non-overlapping. All keys in the ranges
-    /// are guaranteed to be cached.
+    /// are guaranteed to be cached, which means they should be [`Sentinelled::Normal`]s.
```

Comment:
Out of scope for this PR, but suggest giving the sorted ranges their own type.

Reply:
Indeed.

Reply:
Introduced a new type `AffectedRange` for this.